projects/fal/tests/test_stability.py

```diff
@@ -1,17 +1,27 @@
 from __future__ import annotations
 
+import subprocess
 from contextlib import suppress
 
 import fal
 import pytest
 from fal.api import FalServerlessError, Options
 from fal.container import ContainerImage
 from fal.toolkit.file.file import File
+from isolate.backends.common import active_python
 from pydantic import __version__ as pydantic_version
 
 PACKAGE_NAME = "fall"
 
 
+def git_revision_short_hash() -> str:
+    return (
+        subprocess.check_output(["git", "rev-parse", "--short", "HEAD"])
+        .decode("ascii")
+        .strip()
+    )
```

Comment:
fal sha doesn't have much to do with this though. Maybe just use a random string to force building?

Reply:
I didn't want to make it build the environment all the time, actually. I wanted it to be stable within this set of changes.

Reply:
@kakkoyun this is fragile, doesn't take dirty changes into account and also cwd-dependant.

Fal changes should not matter for the building, right? If we are not testing the building here, then lets just embrace the cached images maybe?